android/src/main/java/com/oktareactnative/HttpClientImpl.java

```diff
@@ -12,166 +12,161 @@
 
 package com.oktareactnative;
 
-import android.annotation.SuppressLint;
+import static com.okta.oidc.net.ConnectionParameters.USER_AGENT;
```

Comment:
The new implementation is mostly taken from here: https://github.com/okta/okta-oidc-android/blob/ce298473dc2b1e2eaced212c98f3d3acee636f5c/app/src/main/java/com/okta/oidc/example/OkHttp.java
There are a few changes to the reference implementation, namely customizable timeouts, and user-agent response headers as required by this SDK